google/cloud/storage/internal/hash_function.cc

```diff
@@ -45,7 +45,7 @@ std::unique_ptr<HashFunction> CreateHashFunction(
   if (request.RequiresRangeHeader()) return CreateNullHashFunction();
   return CreateHashFunction(
       request.GetOption<DisableCrc32cChecksum>().value_or(false),
-      request.GetOption<DisableMD5Hash>().value_or(true));
+      request.GetOption<DisableMD5Hash>().value_or(false));
```

Comment:
This default changed from `true` to `false`. Is this an intentional change in behavior?

Reply:
Should not be.  The default constructor for `DisableMD5Hash` sets the value to `true`.  The other constructors either set it to either `true` or `false`, the `_or()` branch is never taken.  It is confusing, but I thought it was less confusing than using `.value_or(false)` for CRC32C and `.value_or(true)` for MD5.  I can change it if you prefer, I think both can be messy.

Reply:
No, it's fine as is. Thanks.